### PR TITLE
Fix permissions issue

### PR DESCRIPTION
**Pull request by fabeschan**

### CODE REVIEW

File: codalab/rest/util.py [hunk 1]
```diff
@@ -182,15 +182,18 @@ def get_group_info(group_spec, need_admin, access_all_groups=False):
...
+    if is_root_user or access_all_groups:
+        # note: this object will NOT contain the 'is_admin' column
+        group_info = unique_group(local.model, group_spec, user_id=None)
+    else:
+        # note: this object will contain the 'is_admin' column
+        group_info = unique_group(local.model, group_spec, user_id=user_id)
 
     # If not root and need admin access, but don't have it, raise error.
-    if user_id and need_admin and not group_info['is_admin'] and user_id != group_info['owner_id']:
+    if not is_root_user and need_admin and 'is_admin' in group_info and not group_info['is_admin']:
```
Comment (percyliang) [inline codalab/rest/util.py]: can this be `group_info.get('is_admin') == False`?

File: codalab/rest/util.py [hunk 2]
```diff
@@ -182,15 +182,18 @@ def get_group_info(group_spec, need_admin, access_all_groups=False):
...
     """
     user_id = request.user.user_id
+    is_root_user = (user_id == local.model.root_user_id)
 
     # If we're root, then we can access any group.
-    if user_id == local.model.root_user_id or access_all_groups:
-        user_id = None
-
-    group_info = unique_group(local.model, group_spec, user_id)
+    if is_root_user or access_all_groups:
```
Comment (percyliang) [inline codalab/rest/util.py]: This section looks pretty tricky...can you write more documentation about what's going on?

**Review by percyliang — APPROVED** [review]
Great.  Could we add tests?

**Comment by fabeschan** [comment]
Yup, just need to figure out a nice way to interact with user groups from two or more different accounts

**Comment by fabeschan** [comment]
Which will be deferred until https://github.com/codalab/codalab-cli/issues/775 is done :)

**Pull request merged**